helm-charts/splunk-otel-collector/templates/deployment-cluster-receiver.yaml

```diff
@@ -1,7 +1,12 @@
 {{ $clusterReceiver := fromYaml (include "splunk-otel-collector.clusterReceiver" .) }}
 {{- if and $clusterReceiver.enabled (eq (include "splunk-otel-collector.metricsEnabled" .) "true") }}
 apiVersion: apps/v1
-kind: Deployment
+{{/*
```

Comment:
Try this to avoid the added blank lines in the rendered manifests: `{{-/*`

Reply:
Thanks, didn't realize whitespace was introduced by comments.